application/models/Iota.php

```diff
@@ -135,34 +128,41 @@ function fetchIota($postdata) {
 			$sql .= " and exists (select 1 from " . $this->config->item('table_name') . " where station_id in (". $location_list . ") and col_iota = iota.tag";
 
 			if ($postdata['mode'] != 'All') {
-				$sql .= " and (col_mode = '" . $postdata['mode'] . "' or col_submode = '" . $postdata['mode'] . "')";
+				$sql .= " and (col_mode = ? or col_submode = ?)";
+				$binding[] = $postdata['mode'];
+				$binding[] = $postdata['mode'];
 			}
 
 			if ($postdata['band'] != 'All') {
 				if ($postdata['band'] == 'SAT') {
-					$sql .= " and col_prop_mode ='" . $postdata['band'] . "'";
+					$sql .= " and col_prop_mode = ?";
```

Comment:
We could hardcode `SAT` here as it can't have another value. But this also works with a bind.

Reply:
Either way works for sure :)